DuckDuckGo/SwipeTabsCoordinator.swift

```diff
@@ -308,14 +308,15 @@ extension SwipeTabsCoordinator: UICollectionViewDataSource {
 
         if !isEnabled || tabsModel.currentIndex == indexPath.row {
             cell.omniBar = coordinator.omniBar
+            cell.setNeedsUpdateConstraints()
```

Comment:
Could just move this outside of the if block since it happens in both conditions?

Reply:
https://github.com/duckduckgo/iOS/pull/3473/commits/68b3bb3fe73618f192752a187802039521a328fc